### DISCUSSION TITLE
How to translate needs title?

### DISCUSSION THREAD

**Discussion by tastenmo** [useblocks/sphinx-needs#1032] [Q&A]
Hi all,

I recently started using needs. I need a multilanguage document. But I cant find a way to get the needs title translated. 

The translation of the content works fine, but even the option 'needs_title_from_content' the title is not translated.

Is it a bug or is there a workaround?

**Comment by danwos** [comment]
Hi @tastenmo,

Ohh this is the first time we have encountered translation requirements in Sphinx-Needs.
So I'm afraid that this is not supported right now.

I also have nearly zero multilanguage experience with Sphinx.
Do you know what is needed for an extension to support this?

**Reply by chrisjsewell** [reply]
FYI, I have a decent knowledge of translation (in fact I just caught a bug in sphinx lol https://github.com/sphinx-doc/sphinx/issues/11689)
